.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: Publish Python 🐍 distribution 📦 to PyPI
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v2
+
+      - name: Build distribution
+        uses: uclahs-cds/tool-pypi-publish-action/build-distribution@a407d50
```

Comment:
Does it work by adding commit ID after the subdirectory of the repo?

Reply:
I think so, but you might need the full commit SHA.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses

They don't show specifically using a "public action in a subdirectory" with a commit SHA, but they do show, separately, both `actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3` and `actions/aws/ec2@main`.

Reply:
Cool! Just updated